examples/notepad/index.html

```diff
@@ -49,94 +49,102 @@ <h1>The Pad module</h1>
+      <ul>
+        <li>Use the store and fetcher to fetch the notepad document.</li>
+        <li>Provide the full path of the pad document you are fetching.</li>
+        <li>Provide the URI of the author/editor of the document.</li>
+        <li>Provide the full path of the subject for this pad. As you can see in the example below it is the same
+          as the pad, except you also need to include '#this'.
+        </li>
+        <li>Optional: Provide a StatusArea property (HTMLDIVElement) to display error messages</li>
+      </ul>
     </p>
+
     <script id='script-notepad'>
       window.addEventListener('DOMContentLoaded', async (event) => {
-        const pad = UI.rdf.namedNode('https://example-user.inrupt.net/public/example-notepad/index.ttl#this')
-        const me = new UI.rdf.NamedNode('https://example-user.inrupt.net/profile/card#me')
-        const subject = UI.store.sym('https://test.test#')
-        options = {}
+        const kb = UI.store
+        const fetcher = UI.rdf.fetcher(kb)
+        const pad = UI.rdf.namedNode('https://sharonstrats.inrupt.net/public/edu.mit.solid.pane.pad/id1585026584938/index.ttl')
+        const me = new UI.rdf.NamedNode('https://sharonstrats.inrupt.net/profile/card#me')
+        const subject = UI.store.sym('https://sharonstrats.inrupt.net/public/edu.mit.solid.pane.pad/id1585026584938/index.ttl#this')
+        const options = { exists: true }
 
+        await fetcher.load(pad.doc()) // if you don't have this you will get an inconsistency error
         const result = UI.pad.notepad(document, pad.doc(), subject, me, options)
         document.querySelector('#div-notepad').appendChild(result)
       })
     </script>
     <pre id='viewSource-notepad'></pre>
     <div id='div-notepad'></div>
-  </article>
 
-  <article id='manageParticipation'>
-
-    <h2>
-      <a href='#manageParticipation'>
-        <code>manageParticipation</code>: Creates the table to display the participants
-      </a>
-    </h2>
-    <p></p>
-    <script id='script-manageParticipation'>
-      window.addEventListener('DOMContentLoaded', (event) => {
-        const me = new UI.rdf.NamedNode('https://example-user.inrupt.net/profile/card#me')
-        const structure = document.createElement('div')
-        const container = document.createElement('div')
-        const subject = UI.store.sym('https://test.test#')
-        const pad = UI.rdf.namedNode('https://example-user.inrupt.net/public/example-notepad/index.ttl#this')
-        options = { statusArea: container }
-        // const result = UI.pad.manageParticipation(document, structure, pad.doc(), subject, me, options)
-        //  document.querySelector('#div-manageParticipation').appendChild(result)
-      })
-    </script>
-    <pre id='viewSource-manageParticipation'></pre>
-    <div id='div-manageParticipation'></div>
+    In order to create a new notepad you need to change three things:
+    <ul>
+      <li>Provide user authentication. You can do this by .... </li>
```

Comment:
... by including solid-auth-client as explained in https://solidproject.org/for-developers/apps/first-app/1-authentication